webpack.config.js

```diff
@@ -68,13 +68,13 @@ const WEBPACK_CONFIG = {
         }],
         proxy: {
             "/yamcs-proxy/*": {
-                target: "http://localhost:8090/",
+                target: "http://127.0.0.1:8090/",
                 secure: false,
                 changeOrigin: true,
                 pathRewrite: {'^/yamcs-proxy/': ''}
             },
             "/yamcs-proxy-ws/*": {
-                target: "ws://localhost:8090/api/websocket",
+                target: "ws://127.0.0.1:8090/api/websocket",
```

Comment:
Same question here

Reply:
@scottbell our webpack-dev-server proxy is not working in CI https://github.com/akhenry/openmct-yamcs/issues/191

Reply:
We think it's due to us not waiting for YAMCS to come up before starting tests